### PR TITLE
change user constructor

### PR DESCRIPTION
**Pull request by RoxaneBurri**

### CODE REVIEW

File: app/src/main/java/com/sdp/movemeet/backend/serialization/UserSerializer.java [hunk 1]
```diff
@@ -15,6 +16,10 @@
     public static final String PHONE_KEY = "phone";
     // The key used to access the description attribute of a serialized User
     public static final String DESCRIPTION_KEY = "description";
+    // The key used to access the created activities attribute of a serialized User
+    private static  final String CREATED_ACTIVITIES_KEY = "createdActivity";
```
Comment (Antho1426) [inline app/src/main/java/com/sdp/movemeet/backend/serialization/UserSerializer.java:20]: Nice! Thank you for taking our conversation into account and adding this field in the `User` objects! This way, as suggested a few weeks ago by @maximilian1064, we will be able to not only completely delete the user from Firebase but also the "Firebase Firestore documents" of the activities he created as an organiser! The "delete account" feature will hence make much more sense!

File: app/src/main/java/com/sdp/movemeet/models/User.java [hunk 2]
```diff
@@ -17,40 +16,45 @@
     private String idUser;
     private String imageId;
     private String documentPath;
+    private ArrayList<String> createActivitiesId;
```
Comment (Antho1426) [inline app/src/main/java/com/sdp/movemeet/models/User.java]: Could you just please rename the name of this ArrayList from createActivitiesId to createdActivitiesId? I think that the missing "d" is a typo here...

File: app/src/main/java/com/sdp/movemeet/models/User.java [hunk 3]
```diff
@@ -111,11 +118,25 @@ public String getDocumentPath() {
         return this.documentPath;
     }
 
+    /**
+     * @return the list of activities created by the user
+     */
+    public ArrayList<String> getCreateActivitiesId() {
```
Comment (Antho1426) [inline app/src/main/java/com/sdp/movemeet/models/User.java]: Here as well, I would rather name this getter "getCreatedActivitiesId" (with a "d").

File: app/src/main/java/com/sdp/movemeet/models/User.java (hunk 2 above)
Comment (RoxaneBurri) [reply app/src/main/java/com/sdp/movemeet/models/User.java]: Yes I did it in my last commit !

**Review by bitshrine — APPROVED** [review]

File: app/src/main/java/com/sdp/movemeet/models/User.java (hunk 2 above)
Comment (Antho1426) [reply app/src/main/java/com/sdp/movemeet/models/User.java]: Nice thank you!

**Review by Antho1426 — APPROVED** [review]

**Pull request merged**